ui/src/app/pages/admin/admin-user-profile.tsx

```diff
@@ -300,33 +301,47 @@ interface TableRow {
 const AccessModuleTable = (props: AccessModuleTableProps) => {
   const { updatedProfile } = props;
 
-  const tableData: TableRow[] = accessModulesForTable.map((moduleName) => {
-    const { adminPageTitle, bypassable } = getAccessModuleConfig(moduleName);
-
-    return {
-      moduleName: adminPageTitle,
-      moduleStatus: displayModuleStatus(props.updatedProfile, moduleName),
-      completionDate: displayModuleCompletionDate(updatedProfile, moduleName),
-      expirationDate: displayModuleExpirationDate(updatedProfile, moduleName),
-      accessTierBadge: displayTierBadgeByRequiredModule(
-        props.updatedProfile,
-        moduleName
-      ),
-      bypassToggle: bypassable && (
-        <ToggleForModule moduleName={moduleName} {...props} />
-      ),
-    };
-  });
+  const tableData: TableRow[] = fp.flatMap((moduleName) => {
+    const { adminPageTitle, bypassable, isEnabledInEnvironment } =
+      getAccessModuleConfig(moduleName);
+
+    return isEnabledInEnvironment
+      ? [
```

Comment:
return an array of size 1 if enabled, none if not: then `flatMap` combines these into a single array

Reply:
👍🏻